pom.xml

```diff
@@ -258,7 +258,7 @@
           -->
           <instructions>
             <Bundle-SymbolicName>com.google.cloud.tools.appengine</Bundle-SymbolicName>
-            <Bundle-RequiredExecutionEnvironment>JavaSE-1.7</Bundle-RequiredExecutionEnvironment>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
```

Comment:
@elharo @briandealwis I don't really know what any of this is, but I assume I need to do this?

Reply:
:+1: